### ISSUE TITLE
Cannot visualize working_with_data.bal in VSCode

### ISSUE THREAD

**Issue by homersimpsons** [ballerina-guides/integration-samples#16]
**Description:**
The code in https://github.com/ballerina-guides/integration-samples/blob/main/working_with_data/working_with_data.bal cannot be visualized in VSCode.

This is caused by the following line:
https://github.com/ballerina-guides/integration-samples/blob/7eb6c8884e3db8ec2821dca82c018251450083f9/working_with_data/working_with_data.bal#L15

In fact doing this diff make it works again:
```diff,ballerina
- Country[] countries = check diseaseEp->/covid\-19/countries;
+ Country[] countries = check diseaseEp->get("/covid-19/countries");
```

I think this should be fixed because this is showcased on the home page for Ballerina: https://ballerina.io/

**Suggested Labels:**
bug,documentation,Priority/High

**Suggested Assignees:**
N/A

**Affected Product Version:** N/A

**OS, DB, other environment details and versions:**
```sh
❯ bal version
Ballerina 2201.5.0 (Swan Lake Update 5)
Language specification 2022R4
Update Tool 1.3.14
```

VSCode extension version: v4.2.0

**Steps to reproduce:**
1. Setup your local environment with Ballerina's VSCode extension
2. Create a new project
3. Copy/Paste the code from https://github.com/ballerina-guides/integration-samples/blob/main/working_with_data/working_with_data.bal
4. Use the "Visualize" Code Lens

Notice that it is stuck in "Loading" state.

**Related Issues:** N/A

**Suggested fix:**

Either:
- Do not showcase this functionality and use `->get("...")`
- Fix the "Visualization" option, note that I do not know how hard it is to do so and it may be pretty easy to do with a quick desugar of this syntax.

**Comment by MaryamZi** [comment]
This seems to have been fixed in version 4.2.2. Please feel free to reopen if you can still observe the same behaviour.

**Comment by homersimpsons** [comment]
I confirm this is working, thank you
